instrumentation/redis/test/opentelemetry/instrumentation/redis_client_test.rb

```diff
@@ -157,9 +183,10 @@ def redis_with_auth(redis_options = {})
 
     it 'records net.peer.name and net.peer.port attributes' do
       expect do
-        Redis.new(host: 'example.com', port: 8321, timeout: 0.01).auth(password)
-      end.must_raise Redis::CannotConnectError
+        redis_with_auth(host: 'example.com', port: 8321, timeout: 0.01)
+      end.must_raise RedisClient::CannotConnectError
 
+      skip('FIXME: what is this intended to test?')
```

Comment:
I missed this `skip` during my last review. 

I think what the intent was was to ensure the span had the `net.peer.*` attributes where captured during a network failure to execute the `auth` command. 

Are there changes in how the driver works that would make this impossible to do?

Reply:
Ah yes, forgot to follow up on that one. RedisClient tries to do the connect before any of the middlewares are invoked, so we don't get a span here under that implementation. I don't think it's within the scope of the PR to change that behavior (previously, the test was erroneously running the RedisV4 client instrumentation), but I didn't want to complete clobber the expectations (these still exist in the other test file). I can clarify the comment so that if that span _is_ ever added for RedisClient, it should have those attributes.

Reply:
I see. In that case then I think it's fine to delete the test since we won't be able to get a span generated from it.